src/views/torrents.html

```diff
@@ -163,6 +163,10 @@ <h2 class="ui center aligned icon header">
         <i class="ui orange arrow up icon"></i>
         {{ totalUploadSpeed | speed }} <span ng-if="totalUploaded">({{ totalUploaded | bytes:2 }})</span>
       </div>
+      <div class="free-disk-space">
```

Comment:
Might need an `ng-if` here for when this info is not available (which will be most clients, except qBittorrent)